### PR TITLE
feature/JU-1224 Added ecs

### PR DESCRIPTION
**Pull request by hreddy-ch**
terraform for service third-party-test-harness

### CODE REVIEW

**Review by bcullerton — COMMENTED** [review]
See slack comment. I'm not sure this is needed

File: terraform/groups/ecs-service/main.tf [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  # ECS Task container health check
+  use_task_container_healthcheck    = true
+  healthcheck_path                  = local.healthcheck_path
+  healthcheck_matcher               = local.healthcheck_matcher
+  health_check_grace_period_seconds = 180
+
+  # Docker container details
+  docker_registry   = var.docker_registry
+  docker_repo       = local.docker_repo
+  container_version = var.alpha_key_service_version
```
Comment (vaibhavcompanieshouse) [inline terraform/groups/ecs-service/main.tf]: is it okay to use name of alpha_key_version ?

File: terraform/groups/ecs-service/variables.tf [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  default     = "SSM_VERSION_"
+}
+
+variable "use_set_environment_files" {
+  type        = bool
+  default     = true
+  description = "Toggle default global and shared environment files"
+}
+
+variable "alpha_key_service_version" {
```
Comment (vaibhavcompanieshouse) [inline terraform/groups/ecs-service/variables.tf]: Typo here

File: terraform/groups/ecs-service/main.tf [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  task_environment            = local.task_environment
+  task_secrets                = local.task_secrets
+  app_environment_filename    = local.app_environment_filename
+  use_set_environment_files   = local.use_set_environment_files
+
+}
+
+
+module "secrets" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.293"
```
Comment (vaibhavcompanieshouse) [inline terraform/groups/ecs-service/main.tf]: might be useful to be 296

**Review by vaibhavcompanieshouse — APPROVED** [review]
Resolved done

File: terraform/groups/ecs-service/variables.tf (hunk 2 above)
Comment (hreddy-ch) [reply terraform/groups/ecs-service/variables.tf]: fixed - feature/JU-1224 typo1

File: terraform/groups/ecs-service/main.tf (hunk 3 above)
Comment (hreddy-ch) [reply terraform/groups/ecs-service/main.tf]: fixed - feature/JU-1224 updating secrets version

**Pull request merged**